packages/bluebird/src/semantic-ui-react/_dropdown.scss

```diff
@@ -0,0 +1,1513 @@
+// .ui.dropdown {
+//   cursor: pointer;
+//   position: relative;
+//   display: inline-block;
+//   outline: none;
+//   text-align: left;
+//   -webkit-transition: width 0.1s ease, -webkit-box-shadow 0.1s ease;
+//   transition: width 0.1s ease, -webkit-box-shadow 0.1s ease;
+//   transition: box-shadow 0.1s ease, width 0.1s ease;
+//   transition: box-shadow 0.1s ease, width 0.1s ease,
+//     -webkit-box-shadow 0.1s ease;
+//   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+// }
+
+// /*******************************
+//               Content
+//   *******************************/
+
+// /*--------------
+//         Menu
+//   ---------------*/
+
+.ui.dropdown .menu {
+  //   cursor: auto;
+  //   position: absolute;
+  //   display: none;
+  //   outline: none;
+  //   top: 100%;
+  //   min-width: -webkit-max-content;
+  //   min-width: -moz-max-content;
+  //   min-width: max-content;
+  //   margin: 0em;
+  padding: $padding-base-vertical 0;
+  background: $white;
+  //   font-size: 1em;
+  //   text-shadow: none;
+  //   text-align: left;
+  -webkit-box-shadow: $shadow-depth-3;
+  box-shadow: $shadow-depth-3;
+  border: 0;
+  border-radius: $input-border-radius;
+  //   -webkit-transition: opacity 0.1s ease;
+  //   transition: opacity 0.1s ease;
+  //   z-index: 11;
+  //   will-change: transform, opacity;
+}
+// .ui.dropdown .menu > * {
+//   white-space: nowrap;
+// }
+
+// /*--------------
+//     Hidden Input
+//   ---------------*/
+
+// .ui.dropdown > input:not(.search):first-child,
+// .ui.dropdown > select {
+//   display: none !important;
+// }
+
+// /*--------------
+//    Dropdown Icon
+//   ---------------*/
+
+.ui.dropdown > .dropdown.icon {
+//   position: relative;
+//   width: auto;
+//   font-size: 0.85714286em;
+  margin: 0em 0em 0em 1em;
```

Comment:
`1em` seems non-standard. Should this value come from variables?

Reply:
@kirbyyardley explain yourself

Reply:
This was correct. It also helped me see another issue, which I'm pushing up now.